src/test/ui/resolve/issue-14254.stderr

```diff
@@ -2,7 +2,7 @@ error[E0425]: cannot find function `baz` in this scope
   --> $DIR/issue-14254.rs:29:9
    |
 29 |         baz();
-   |         ^^^ did you mean `self.baz(...)`?
+   |         ^^^ help: did you mean `self.baz`
```

Comment:
Ok. I implemented this, but the code complexity goes way up. I need to find the correct `{}`. I'll also need to allow `{{` to escape opening brackets. I need to allow some way to specify a replacement like "the following", .... 

How about my [first variant suggested here](https://github.com/rust-lang/rust/pull/40851#issuecomment-290641507)?

> add a new enum so the lint author can choose a message

```rust
enum SuggestionMsg {
    DidYouMean,
    Try,
    Custom {
        label: String, // msg to display as label
        help: String, // msg to display as help
    },
}
```

cc @nrc this touches on prepending `help:` before suggestions displayed as labels